### PR TITLE
Use in-memory database during timing tests

### PR DESCRIPTION
**Pull request by sebastiencs**
This should fix tests in CI builds

### CODE REVIEW

**Comment by tizoc** [comment]
@sebastiencs ok, cool. I see this one includes the key deduplication, is that part ready too?

**Comment by sebastiencs** [comment]
@tizoc Yes this is ready

**Comment by tizoc** [comment]
Ok, cool, waiting for the tests to finish and then I will merge.

File: tezos/timing/src/schema_stats.sql [hunk 1]
```diff
@@ -22,16 +22,22 @@ CREATE TABLE IF NOT EXISTS contexts (
   hash TEXT UNIQUE
 );
 
+CREATE TABLE IF NOT EXISTS keys (
+  id INTEGER PRIMARY KEY AUTOINCREMENT,
+  key TEXT NOT NULL UNIQUE
```
Comment (tizoc) [inline tezos/timing/src/schema_stats.sql:27]: @sebastiencs did you notice any performance difference with this implementation? We are missing an index on this one, without it I think things will start to slow down after a bunch of keys have been added

Comment (sebastiencs) [reply tezos/timing/src/schema_stats.sql:27]: My understanding is that `UNIQUE` automatically makes an index on that column.
I tried to add a `CREATE INDEX key_idx ON keys(key);` but it makes a error saying that the index already exist.

Comment (tizoc) [reply tezos/timing/src/schema_stats.sql:27]: @sebastiencs oh, in sqlite I think `UNIQUE` already creates an index, let me check.

**Pull request merged**